### ISSUE TITLE
Setup failed for custom integration blueiris: Unable to import component: cannot import name 'async_get_registry'

### ISSUE THREAD

**Issue by cheechm13** [elad-bar/ha-blueiris#208]
It looks like after Home Assistant Core 2023.5.0 and after they have depracated async_get_registry and replaced it with async_get.

See here: https://community.home-assistant.io/t/error-async-get-registry-after-core-2023-5-0/567951/6

And here: https://github.com/garbled1/homeassistant_ecowitt/pull/153

Need to find where used and replace to get Blue Iris working again after core update.

**Comment by kramttocs** [comment]
I'll take a look.
Did you have it installed prior to upgrading HA?
I am on the latest HA and mine is still working so odd if yours quit.

**Comment by cheechm13** [comment]
> I'll take a look. Did you have it installed prior to upgrading HA? I am on the latest HA and mine is still working so odd if yours quit.

After searching through the .py files for the async_get_registry call out, I happend upon the changelog and the latest change says this has been fixed. I looked at HACS and for some reason it wasn't popping up as an updated was needed.

I did a manual update and then HA restart and now it is updated and working fine.

Not sure what caused the integration to not trigger an update/restart previously, but all is well now.
